Userland/Libraries/LibJS/Runtime/PromiseReaction.h

```diff
@@ -20,22 +20,22 @@ struct PromiseCapability {
-    ({                                                                                                                  \
-        auto _temporary_try_or_reject_result = (expression);                                                            \
-        /* 1. If value is an abrupt completion, then */                                                                 \
-        if (_temporary_try_or_reject_result.is_error()) {                                                               \
-            vm.clear_exception();                                                                                       \
-                                                                                                                        \
-            /* a. Perform ? Call(capability.[[Reject]], undefined, « value.[[Value]] »). */                           \
-            TRY(vm.call(*capability.reject, js_undefined(), *_temporary_try_or_reject_result.release_error().value())); \
-                                                                                                                        \
-            /* b. Return capability.[[Promise]]. */                                                                     \
-            return capability.promise;                                                                                  \
-        }                                                                                                               \
-                                                                                                                        \
-        /* 2. Else if value is a Completion Record, set value to value.[[Value]]. */                                    \
-        _temporary_try_or_reject_result.release_value();                                                                \
+#define TRY_OR_REJECT(vm, global_object, capability, expression)                                                                        \
+    ({                                                                                                                                  \
+        auto _temporary_try_or_reject_result = (expression);                                                                            \
+        /* 1. If value is an abrupt completion, then */                                                                                 \
+        if (_temporary_try_or_reject_result.is_error()) {                                                                               \
+            vm.clear_exception();                                                                                                       \
+                                                                                                                                        \
+            /* a. Perform ? Call(capability.[[Reject]], undefined, « value.[[Value]] »). */                                           \
```

Comment:
clang-format does not care that these unicode chars display shorter, just assumes each byte is the same size...